Sources/BranchSDK/Branch.m

```diff
@@ -1920,10 +1922,11 @@ - (void)processNextQueueItem {
 
         if (req) {
 
-            // If tracking is disabled, then do not check for install event.  It won't exist.
+            // If tracking is disabled, then do not check for install event. It won't exist.
             if (!Branch.trackingDisabled) {
                 if (![req isKindOfClass:[BranchInstallRequest class]] && !self.preferenceHelper.randomizedBundleToken) {
                     [[BranchLogger shared] logError:@"User session has not been initialized!" error:nil];
+                    self.networkCount = 0;                    
```

Comment:
Does this need to be a thread safe operation? Can other areas be checking this value at the same time, and executing/not executing an event because of it? cc @NidhiDixit09

Reply:
@gdeluna-branch `networkCount` is defined as a property and its protected using `@synchronized (self)` inside its accessor functions. So its thread safe. 

@nsingh-branch I think we should add this fix for opens also. Its a rare case to happen but technically `else if `  condition has the same issue.